### ISSUE TITLE
Extend createOptions length

### ISSUE THREAD

**Issue by andrusza2** [Azure/iot-edge-virtual-kubelet-provider#26]
The `createOptions` field in the Edge Agent's twin is part of a module's Docker configuration.
It is an optional field and is used to customize the settings of the Docker container.
Its length is limited to 512 bytes and this is often insufficient (in situations such as binding ports or mapping devices to a container). 
This problem has been solved in IoT Edge 1.0.4 release (check https://azure.microsoft.com/en-us/updates/iot-edge1-0-4/) by chunking this field (see  https://github.com/Azure/iotedge/pull/396).
Unfortunately, at the moment it is impossible to use a virtual kubelet for deployment requiring longer `createOptions` field. The attempt of such deployment ends with an error:
```
Microsoft.Azure.VirtualKubelet.Server.Controllers.VirtualKubeletProviderController[100]
2019-04-16T08:25:22.015693561Z       Error
2019-04-16T08:25:22.015723461Z System.ArgumentException: {"Message":"ErrorCode:ArgumentInvalid;Property or Tag value should be maximum 512 bytes. (...) }
```
Is there any way to use a virtual kubelet in this case?

**Comment by veyalla** [comment]
Thanks for the report @andrusza2! You're right, it will take some work in the translation to enable. I'll add it to our backlog. FYI @avranju

**Comment by neumanndaniel** [comment]
Solved PR #27

**Comment by veyalla** [comment]
`microsoft/iot-edge-virtual-kubelet-provider:latest` has been updated to contain these fixes. Thanks all!
